.kitchen.yml

```diff
@@ -38,6 +40,24 @@ suites:
           elasticsearch:
             config:
               network.host: 0.0.0.0
+            plugins:
+              lang-python: lang-python
```

Comment:
We should stick to the plugin that you started with and already in the example, kopf.